data/_ui/help.txt

```diff
@@ -172,6 +172,7 @@ help "stranded"
 help "trading"
 	`This is the trading panel. Earn money by buying commodities at a low price in one system, and selling at a higher price elsewhere. To view your map of commodity prices in other systems, press <View star map>. To buy or sell, click on [buy] or [sell], or select a line with the up and down arrows and press "+" or "-" (or Enter and Delete).`
 	`You can buy 5 tons at once by holding down Shift, 20 by holding down Control, 500 by holding down Alt, or larger amounts at a time by holding down two keys at once, or even all three.`
+	`Sell other items in your cargo hold with the buttons to the left. "Sell Outfits" sells any outfitter items in your cargo hold. "Sell Specials" sells materials you mine from asteroids.`
```

Comment:
Minables are "outfitter items", too.
```suggestion
	`Sell other items in your cargo hold with the buttons to the left. "Sell Specials" sells materials you mine from asteroids. "Sell Outfits" sells any other outfitter items in your cargo hold.`
```

Reply:
Special commodities aren't necessarily only mineables. For now though, others are a niche case.